core/dbt/include/global_project/macros/materializations/models/incremental/on_schema_change.sql

```diff
@@ -119,6 +119,11 @@
                 - set the `on_schema_change` config to either append_new_columns or sync_all_columns, depending on your situation.
                 - Re-run the incremental model with `full_refresh: True` to update the target schema.
                 - update the schema manually and re-run the process.
+
+              Extra info for easy anomaly detection:
+                 Source columns not in target: {{ schema_changes_dict['source_not_in_target'] }}
+                 Target columns not in source: {{ schema_changes_dict['target_not_in_source'] }}
+                 New column types: {{ schema_changes_dict['new_target_types'] }}
```

Comment:
@dbeatty10 @dataders Does either of you have thoughts on the specific language here?

Reply:
Something like "Additional troubleshooting context:" could be a drop-in replacement for "Extra info for easy anomaly detection:"

Side note: it's a bit unfortunate that "schema" is both the 2nd level in a three-level `database.schema.relation` hierarchy as well as the result of [DDL](https://en.wikipedia.org/wiki/Data_definition_language) operations on a relation.

Reply:
@dbeatty10 I updated the log message per your request. It looks better